DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc

```diff
@@ -296,9 +298,9 @@ void TrackBuildingAnalyzer::analyze
   GlobalPoint  v(v0.x()-bs.x0(),v0.y()-bs.y0(),v0.z()-bs.z0());
   
   double pt           = sqrt(state.globalMomentum().perp2());
-  double eta          = state.globalMomentum().eta();
-  double phi          = state.globalMomentum().phi();
-  double theta        = state.globalMomentum().theta();
+  double eta          = state.globalPosition().eta();
```

Comment:
POCA is the main issue. we whant to correrate with detector issues, not to physics...
so indeed eta computed from direction was pretty random smearing any real feature in particular at large r.
now detector issue shows up immediately. will show examples tomorrow at trk dpg meeting